discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -133,3 +149,31 @@ def update_user(params: ManageEntityParameters):
     params.add_user_record(user_id, user_record)
 
     return user_record
+
+
+def verify_user(params: ManageEntityParameters):
```

Comment:
does the user signup flow cover send a verify user tx? i guess all create_user txs have is_verified false to start